Dockerfile

```diff
@@ -19,10 +19,11 @@ RUN apt-get update && apt-get install --no-install-recommends -y build-essential
     avrdude \
     && rm -rf /var/lib/apt/lists/*
 
-ENV keyboard=ergodox
-ENV subproject=ez
-ENV keymap=default
+ENV KEYBOARD=ergodox_ez
+ENV KEYMAP=default
```

Comment:
Note that backwards compatibility with scripts that users may have for running QMK in Docker is not broken, as those scripts will continue to use the `edasque/qmk_firmware` image, which still has the old env vars.